archunit/src/test/java/com/tngtech/archunit/lang/conditions/FieldAccessConditionTest.java

```diff
@@ -51,36 +51,33 @@ public void FieldAccessCondition_satisfied_on_both_get_and_set_field() {
         JavaFieldAccess setAccess = accessFromCallerToTargetWithType(SET);
         FieldAccessCondition setFieldCondition = new FieldAccessCondition(
                 target(name(setAccess.getTarget().getName())));
-        assertSatisfiedWithMessage(setFieldCondition, setAccess, "sets");
+        assertSatisfied(setFieldCondition, setAccess);
 
         JavaFieldAccess getAccess = accessFromCallerToTargetWithType(GET);
         setFieldCondition = new FieldAccessCondition(
                 target(name(getAccess.getTarget().getName())));
-        assertSatisfiedWithMessage(setFieldCondition, getAccess, "gets");
+        assertSatisfied(setFieldCondition, getAccess);
     }
 
-    private void assertSatisfiedWithMessage(FieldAccessCondition getFieldCondition, JavaFieldAccess access,
-            String accessDescription) {
+    private void assertSatisfied(FieldAccessCondition getFieldCondition, JavaFieldAccess access) {
         ConditionEvents events = checkCondition(getFieldCondition, access);
-        boolean satisfied = !events.containViolation();
-
-        assertThat(satisfied).as("Events are satisfied").isTrue();
+        assertThat(!events.containViolation()).as("Events are satisfied").isTrue();
```

Comment:
I'd get rid of the negation and stick to the API, which focusses on violations:
```suggestion
        assertThat(events.containViolation()).as("Events are violated").isFalse();
```

Reply:
Absolutely :joy: